src/main/kotlin/race/ui/ResultView.kt

```diff
@@ -11,9 +11,16 @@ class ResultView {
 
         fun printNow(cars: List<Car>) {
             cars.forEach {
+                print("${it.name} : ")
                 IntRange(1, it.position).forEach { _ -> print("-") }
```

Comment:
```suggestion
                repeat(it.position){
                    print("-")
                }
```

`repeat`을 활용하셔도 좋을 것 같아요 :)

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/repeat.html